source/grid/grid_generator.cc

```diff
@@ -777,13 +784,15 @@ namespace GridGenerator
       {
         for (unsigned int j = 0; j < 2; ++j)
           {
-            unsigned int offset = (8 * (c + j)) % (8 * n_cells_toroidal);
+            unsigned int offset =
```

Comment:
```suggestion
            const unsigned int offset =
```

Reply:
done in https://github.com/dealii/dealii/pull/8916/commits/df2226367fedbca8a7dc1829f1db034db63b7554